### ISSUE TITLE
sfdx not automatically updating

### ISSUE THREAD

**Issue by aheber** [forcedotcom/cli#126]
### Summary

It seems that before the ocliff cutover DX was correctly checking for updates. It seems to have lost that functionality somewhere in here. DX is no longer reporting that updates are available or automatically updating itself. We didn't notice for a little while so I don't have specific versions to identify.

Just yesterday (6/27) I ran `sfdx --version` which used to reliably report if a new version was available and it reported nothing. I then ran `sfdx update` and it found and updated to the new version.

### Steps To Reproduce:

0. Confirm that you do NOT have an environment variable `SFDX_AUTOUPDATE_DISABLE` set.
1. Install an older version, either the current binary version available, 7.8.1 or the immediately previous version 7.12.3.
2. Run a variety of DX commands
3. Run `sfdx --version` and note that the version number has not increased

### Expected result

DX should be auto-updating or at least notifying of new versions as it used to.

### Actual result

DX sits at whatever version was most recently installed or manually updated to and doesn't notify the user that an update is available.

### Additional information

![image](https://user-images.githubusercontent.com/2173116/60359196-8f9b3080-9995-11e9-9b8a-01a711365ecb.png)

This was discussed in the Good Day, Sir! Slack team and included other individuals reporting the same: https://gooddaysir.slack.com/archives/C0ZNTTU20/p1561658506020700

To work around this I had to figure out how to check for updates myself and built a notification system to alert me when a new version is available.

**SFDX CLI Version**(to find the version of the CLI engine run sfdx --version):
`sfdx-cli/7.13.0-27dbcb37d3 win32-x64 node-v10.15.3`

**SFDX plugin Version**(to find the version of the CLI plugin run sfdx plugins --core)
```
@oclif/plugin-commands 1.2.2 (core)
@oclif/plugin-help 2.2.0 (core)
@oclif/plugin-not-found 1.2.2 (core)
@oclif/plugin-plugins 1.7.8 (core)
@oclif/plugin-update 1.3.9 (core)
@oclif/plugin-warn-if-update-available 1.7.0 (core)
@oclif/plugin-which 1.0.3 (core)
@salesforce/sfdx-trust 3.0.5 (core)
analytics 1.1.2 (core)
generator 1.1.0 (core)
salesforcedx 46.3.0 (core)
├─ force-language-services 46.8.0 (core)
└─ salesforce-alm 46.7.0 (core)

sfdx-cli 7.13.0 (core)
sfdx-heber 0.0.2 (link) C:\Users\aheber\dev\sfdx-heber
sfdx-typegen 0.6.2
shane-sfdx-plugins 3.19.0
```
**OS and version**:
Windows 10

**Comment by clairebianchi** [comment]
@aheber We currently are having an issue with our installers for Mac due to a migration we just underwent for our release process. We think this has something to do with it. The installers are scheduled to be fixed mid-July, if this does not fix this issue then we will investigate further.

**Comment by mshanemc** [comment]
closed as not reproducible on W-6294054 9/8/2020
